src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/Snapshot/Filters/ImplicitTopLevelDependenciesSnapshotFilter.cs

```diff
@@ -9,11 +9,11 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.Filters
 {
     /// <summary>
-    /// Changes resolved top level project dependencies to unresolved if:
-    ///     - dependent project does not have targets supporting given target framework in current project
-    ///     - dependent project has any unresolved dependencies in a snapshot for given target framework
-    /// This helps to bubble up error status (yellow icon) for project dependencies.
```

Comment:
This comment was completely misleading. It was probably copied from somewhere else.